### PR TITLE
remove userhack

### PR DESCRIPTION
**Pull request by emord**
I think we can get rid of this now that we're in 1.10 https://docs.djangoproject.com/en/1.10/releases/1.10/#abstractuser-username-max-length-increased-to-150

@nickpell @sravfeyn

### CODE REVIEW

**Comment by dimagimon** [comment]
Could not review pull request. It may be too large, or contain no reviewable changes.

**Review by nickpell — APPROVED** [review]

**Review by nickpell — APPROVED** [review]

**Review by sravfeyn — APPROVED** [review]

File: corehq/apps/userhack/admin.py [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-class DDUserNonceAdmin(admin.ModelAdmin):
-    list_display = ('user', 'nonce', 'count', 'last_used_at')
-
-
-class DDPartialDigestAdmin(admin.ModelAdmin):
-    list_display = ('user', 'partial_digest', 'confirmed')
-    search_fields = ('login',)
-
-admin.site.register(UserNonce, DDUserNonceAdmin)
-admin.site.register(PartialDigest, DDPartialDigestAdmin)
```
Comment (snopoke) [inline corehq/apps/userhack/admin.py]: do we want to keep these model's available on the admin UI?

Comment (emord) [inline corehq/apps/userhack/admin.py]: good point. i added these back

**Comment by emord** [comment]
Removed the don't merge label. I think this should be fine

**Pull request merged**